app/pdf-viewer/eaf-pdf-viewer.el

```diff
@@ -119,12 +121,14 @@
     (with-current-buffer (get-buffer-create  eaf-pdf-outline-buffer-name)
       (setq buffer-read-only nil)
       (erase-buffer)
+      (setq outline-regexp "^[[:space:]]*")
       (insert toc)
+      (outline-mode)
       (setq toc (mapcar (lambda (line)
                           (string-to-number (car (last (split-string line " ")))))
                         (butlast (split-string (buffer-string) "\n"))))
       (goto-line (seq-count (apply-partially #'>= page-number) toc))
-      (set (make-local-variable 'eaf-pdf-outline-original-buffer-name) buffer-name)
+      (set (make-local-variable 'eaf-pdf-outline-original-buffer-name) buf)
```

Comment:
this change is correct?

Reply:
Yes, I changed `eaf-pdf-outline-original-buffer-name` to be a buffer object rather than its name. Should've changed the variable name to reflect that. This is important since `eaf-pdf-outline-view` use the buffer object to find `eaf--buffer-id` of the original buffer